cfgov/unprocessed/css/organisms/footer.less

```diff
@@ -25,7 +27,7 @@
 
     &_share-icon-list {
         margin-top: unit( @grid_gutter-width / @base-font-size-px, em );
-        .list__horizontal();
+        .m-list__horizontal();
```

Comment:
Hmmm `o-footer_share-icon-list` appears to be unused. I'll put in a code comment and it can be investigated and deleted in another PR.